tests/command.bats

```diff
@@ -708,6 +708,22 @@ EOF
   unstub docker
 }
 
+@test "Runs BUILDKITE_COMMAND with platform" {
+  export DOCKER_ARCH=$(arch | sed 's/aarch64/arm64/' | sed 's/x86_64/amd64/')
```

Comment:
there is no need to inspect the actual architecture as the command will never be actually executed, you can fix this to whatever value you want as long as it is the same in the stub

Reply:
ah ok, i wasnt sure if it was going to run, and wanted to be sure it was safe wherever it ran, but i can hard code that. 

thanks!